presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -60,45 +63,56 @@
     @BeforeTestWithContext
     public void setup()
     {
-        tableName = "alice_owned_table";
         aliceExecutor = connectToPresto("alice@presto");
         bobExecutor = connectToPresto("bob@presto");
 
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```

Comment:
I just thought that as long as you're fixing this, you could address the other parallelism concerns (since it isn't too much more). A new PR is fine though.